Lib/pyclbr.py

```diff
@@ -193,8 +215,8 @@ def _create_tree(fullmodule, path, fname, source, tree, inpackage):
     """
     f = io.StringIO(source)
 
-    stack = [] # Initialize stack of (class, indent) pairs.
-
+    # stack = []  # Initialize stack of (class, indent) pairs.
+    stack = Stack()  # changing the source code so as to get the ending line
```

Comment:
This comment belongs in a commit message, not in the source code. The old comment is fine.

```suggestion
    stack = Stack()  # Initialize stack of (class, indent) pairs.
```